### PR TITLE
add openjdk to travis

### PR DESCRIPTION
**Pull request by lovepoem**
## What is the purpose of the change

add openjdk to travis

## Brief changelog

.travs.yml

## Verifying this change

CI success

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/incubator-dubbo/tree/master/dubbo-test).
- [ ] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,9 @@ language: java
 sudo: false # faster builds
 
 jdk:
+  - openjdk11
```
Comment (ralf0131) [inline .travis.yml]: Why do we need both? I think we only need to choose between oraclejdk and openjdk.

Comment (lixiaojiee) [reply .travis.yml]: I have the same question.

**Comment by ralf0131** [comment]
Yes, I agree to use openjdk, but why should we keep oraclejdk?

**Comment by lovepoem** [comment]
>  Yes, I agree to use openjdk, but why should we keep oraclejdk?

Or should I send a mail to dev@ list for discussion?

**Comment by tswstarplanet** [comment]
I think if Dubbo can use Oracle JDK freely, we can contains it in travis. Because companies using Oracle JDK may use Dubbo too. We should consider those users.

**Comment by ralf0131** [comment]
> > Yes, I agree to use openjdk, but why should we keep oraclejdk?
> 
> Or should I send a mail to dev@ list for discussion?

I've raised a discussion before, search for `CI failure for Java 11 - switching to openjdk?` I think we can discuss there, or you can discuss by forwarding this message to dev@ mailing list.

**Pull request merged**